project-properties.gradle

```diff
@@ -86,7 +86,8 @@ project.ext {
             (migrationsUrl + '/migrations/96_create_groups_tables.up.sql')             : 'V096__create_groups_tables.sql',
             (migrationsUrl + '/migrations/97_add_groups_uuid.up.sql')                  : 'V097__add_groups_uuid.up.sql',
             (migrationsUrl + '/migrations/98_add_plugin_type.up.sql')                  : 'V098__add_plugin_type.up.sql',
-            (migrationsUrl + '/migrations/106_create_log_type_table.up.sql')           : 'V106__create_log_type_table.sql'
+            (migrationsUrl + '/migrations/106_create_log_type_table.up.sql')           : 'V106__create_log_type_table.sql',
+            (migrationsUrl + '/migrations/107_add_analysis_owner.up.sql')              : 'V107__add_analysis_owner.sql'
```

Comment:
PR’s failing because of this - merging [the migrations PR](https://github.com/reportportal/migrations/pull/390) will fix it.